### PR TITLE
Fix js issues for project dashboards without wiki widget

### PR DESCRIPTION
**Pull request by rliebz**

### CODE REVIEW

File: website/addons/wiki/templates/wiki_widget.mako [hunk 1]
```diff
@@ -12,9 +12,8 @@
 <% import json %>
 <script>
     window.contextVars = $.extend(true, {}, window.contextVars, {
+        wikiWidget: true,
         usePythonRender: ${json.dumps(use_python_render)},
-        urls: {
```
Comment (sloria) [inline website/addons/wiki/templates/wiki_widget.mako]: @rliebz Why was this changed?

Comment (rliebz) [inline website/addons/wiki/templates/wiki_widget.mako]: Because it was the only URL attached to that object on the dashboard page. I can revert that if it's better for the object to be there for future use

Comment (sloria) [inline website/addons/wiki/templates/wiki_widget.mako]: We want to keep the namespacing consistent. Notice that we use `$.extend` to merge this object with the existing context vars. This way, all URLs that are put into contextVars are under the `urls` property.

I went ahead and reverted this during the merge.
